types/dialogify-browser/index.d.ts

```diff
@@ -0,0 +1,15 @@
+// Type definitions for non-npm package dialogify-browser 1.0
+// Project: https://github.com/OneupNetwork/dialogify
+// Definitions by: moontai0724 <https://github.com/moontai0724>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// Minimum TypeScript Version: 4.4
```

Comment:
```suggestion
```
I'll post suggestions to fix tests issue, it's not required for a implementation itself

Reply:
That's awesome, I didn't imagine that before.